libdnf5/repo/repo_sack.cpp

```diff
@@ -468,6 +468,12 @@ void RepoSack::update_and_load_repos(libdnf5::repo::RepoQuery & repos, bool impo
 
 
 void RepoSack::update_and_load_enabled_repos(bool load_system) {
+    static bool called = false;
```

Comment:
This might be a coarse way of solving this. I also considered adding a flag on the class, but I wasn't sure if repositories can only be loaded once per program invocation, or only once for this given `RepoSack` instance. This can definitely be changed